src/commonMain/kotlin/fr/acinq/lightning/channel/Channel.kt

```diff
@@ -369,6 +368,8 @@ sealed class ChannelStateWithCommitments : PersistedChannelState() {
     val isInitiator: Boolean get() = commitments.params.localParams.isInitiator
     val remoteNodeId: PublicKey get() = commitments.remoteNodeId
 
+    fun ChannelContext.channelKeys(): KeyManager.ChannelKeys = commitments.params.localParams.channelKeys(keyManager)
```

Comment:
This method is not strictly necessary, but comes in handy and reduces boiler plate. Unfortunately we can't use it in state `WaitForFundingSigned`, but that's fine.

Note that keys are recomputes every single time. They could easily be cached if performance is an issue.